packages/cli/lib/errorHandlers/overrrideErrors.js

```diff
@@ -0,0 +1,95 @@
+const { isSpecifiedError } = require('@hubspot/local-dev-lib/errors/apiErrors');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+
+const { PLATFORM_VERSION_ERROR_TYPES } = require('../constants');
+const { i18n } = require('../lang');
+const { uiLink } = require('../ui');
+
+const i18nKey = 'cli.lib.errorHandlers.overrideErrors';
+
+function createDashes() {
+  return '-'.repeat(79);
+}
+
+function createPlatformVersionError(subCategory, errorData) {
+  const platformVersion = errorData.context.PLATFORM_VERSION[0];
+  const docsLink = uiLink(
+    i18n(`${i18nKey}.platformVersionErrors.docsLink`),
+    'https://developers.hubspot.com/docs/platform/platform-versioning'
+  );
+  let translationKey;
+
+  if (
+    subCategory === PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_NOT_SPECIFIED
+  ) {
+    translationKey = 'unspecifiedPlatformVersion';
+  } else if (
+    subCategory === PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_RETIRED
+  ) {
+    translationKey = 'platformVersionRetired';
+  } else if (
+    subCategory ===
+    PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_SPECIFIED_DOES_NOT_EXIST
+  ) {
+    translationKey = 'nonExistentPlatformVersion';
+  }
+
+  logger.log(createDashes());
+  logger.log(i18n(`${i18nKey}.platformVersionErrors.header`));
+  logger.log(
+    i18n(`${i18nKey}.platformVersionErrors.${translationKey}`, {
+      platformVersion,
+    })
+  );
+  logger.log(i18n(`${i18nKey}.platformVersionErrors.updateProject`));
+  logger.log(
+    i18n(`${i18nKey}.platformVersionErrors.betaLink`, {
+      docsLink,
+    })
+  );
+  logger.log(createDashes());
+}
+
+function overrideErrors(err) {
+  if (
+    isSpecifiedError(err, {
```

Comment:
My instinct was to create a helper function to determine the three platform version specified errors, since `isSpecifiedError` is a little clunky to write out every time. But I realized that I would still need to include the helper function three times and include the subcategory each time. So I just left as is with `isSpecifiedError`.